dts/bindings/pinctrl/nuvoton,npcm-pinctrl.yaml

```diff
@@ -0,0 +1,164 @@
+    is also included.
+
+  include:
+    - name: pincfg-node.yaml
+      property-allowlist:
+        - bias-pull-down
+        - bias-pull-up
+
+  properties:
+    nuvoton,pinmux-group-def:
+      type: boolean
+      description: Indicates that this node is a pinmux group definition.
+    nuvoton,pin-reg:
+      type: array
+      description: |
+        The pinmux group definition property.
+        The alternate pin registers and their bits. There are a total of 5 registers.
+        The register order is from 4 to 0 for convenience in setting the alternate function config
+        value, and the maximum bits to configure are 5 (the alternate function config value is 0x1F
+        when all register bits are set).
+        Set NPCM_PIN_REG_NULL for an empty register.
+    nuvoton,alt:
+      type: array
+      description: |
+        The pinmux group definition property.
+        The alternate pin selection configurations consist of a total of 6 groups.
+        The group order is from 0 to 5, and the bit N value in each group is for
+        register N's bit value.
+        Set NPCM_PIN_CFG_NULL for an empty configuration.
+    nuvoton,id:
+      type: array
+      description: |
+        The pinmux group definition property.
+        The pin's index. The name of this property follows the group's order.
+    nuvoton,lv-reg:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The low-voltage level register and its bit.
+    nuvoton,lv-def:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The default voltage level in hardware of supported alt functions. Set with either
+        NPCM_IO_VOLT_LV_1V8 or NPCM_IO_VOLT_LV_3V3.
+    nuvoton,lv-sup:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The flag of low-voltage level support. The bit N value represents that group N supports
+        low-voltage level.
+    nuvoton,pupd-config-extend:
+      type: boolean
+      description: |
+        The peripheral pull-up/down definition property.
+        Indicates that there are two groups of peripheral pull-up/down configurations. The default
+        group number is 1.
+    nuvoton,pupd-reg:
+      type: array
+      description: |
+        The peripheral pull-up/down definition property.
+        The peripheral pull-up/down register and its bit. There are a total of 2 registers if
+        property pupd-config-extend is set.
+    bias-pull-up:
+      type: boolean
+      description: |
+        The peripheral pull-up/down definition property.
+        The default peripheral pull-up in hardware of supported alt functions.
+    bias-pull-down:
+      type: boolean
+      description: |
+        The peripheral pull-up/down definition property.
+        The default peripheral pull-down in hardware of supported alt functions.
```

Comment:
please, do not redefine standard properties.

Reply:
Removed.